docker_files/cell-ranger-count-2.0.2

```diff
@@ -0,0 +1,12 @@
+# Single Cell Portal Cell Ranger Count (2.0.2)
```

Comment:
Is this Docker file separate from the toy data script updates?  This doesn't add a Docker dependency for the running that script, right?

Reply:
Yeah it is a separate change